openedx_events/content_authoring/signals.py

```diff
@@ -200,13 +202,58 @@
     }
 )
 
+# .. event_type: org.openedx.content_authoring.content.object.associations.changed.v1
+# .. event_name: CONTENT_OBJECT_ASSOCIATIONS_CHANGED
+# .. event_description: emitted when an object's associations are changed, e.g tags, collections
+# .. event_data: ContentObjectData
+CONTENT_OBJECT_ASSOCIATIONS_CHANGED = OpenEdxPublicSignal(
+    event_type="org.openedx.content_authoring.content.object.associations.changed.v1",
+    data={
+        "content_object": ContentObjectChangedData
+    }
+)
+
 # .. event_type: org.openedx.content_authoring.content.object.tags.changed.v1
 # .. event_name: CONTENT_OBJECT_TAGS_CHANGED
 # .. event_description: emitted when an object's tags are changed
+#    DEPRECATED: please use CONTENT_OBJECT_ASSOCIATIONS_CHANGED instead.
 # .. event_data: ContentObjectData
 CONTENT_OBJECT_TAGS_CHANGED = OpenEdxPublicSignal(
     event_type="org.openedx.content_authoring.content.object.tags.changed.v1",
     data={
         "content_object": ContentObjectData
     }
 )
+
+# .. event_type: org.openedx.content_authoring.content.library.collection.created.v1
+# .. event_name: LIBRARY_COLLECTION_CREATED
+# .. event_description: emitted when a content library collection is created
+# .. event_data: LibraryCollectionData
+LIBRARY_COLLECTION_CREATED = OpenEdxPublicSignal(
+    event_type="org.openedx.content_authoring.content.library.collection.created.v1",
+    data={
+        "library_collection": LibraryCollectionData
+    }
+)
+
+# .. event_type: org.openedx.content_authoring.content.library.collection.updated.v1
+# .. event_name: LIBRARY_COLLECTION_UPDATED
+# .. event_description: emitted when when a content library collection is updated
+# .. event_data: LibraryCollectionData
+LIBRARY_COLLECTION_UPDATED = OpenEdxPublicSignal(
+    event_type="org.openedx.content_authoring.content.library.collection.updated.v1",
+    data={
+        "library_collection": LibraryCollectionData
+    }
+)
+
+# .. event_type: org.openedx.content_authoring.content.library.collection.deleted.v1
+# .. event_name: LIBRARY_COLLECTION_DELETED
+# .. event_description: emitted when an when a content library collection is deleted
+# .. event_data: LibraryCollectionData
+LIBRARY_COLLECTION_DELETED = OpenEdxPublicSignal(
+    event_type="org.openedx.content_authoring.content.library.collection.deleted.v1",
```

Comment:
```suggestion
    event_type="org.openedx.content_authoring.content_library.collection.deleted.v1",
```

Should we just have `content_library` as the subdomain, [Ref ADR](https://github.com/open-craft/openedx-events/blob/18a35440321e33b30711afcff6944d822df200fb/docs/how-tos/creating-new-events.rst#L25)

Reply:
Oh good catch! I agree we should use the subdomain used in edx-platform, so fixed with [5e89cbe](https://github.com/openedx/openedx-events/pull/386/commits/5e89cbe11e2794c7e444320dbca3954fdfcede3e).